### ISSUE TITLE
Allow configuration of frontmatter

### ISSUE THREAD

**Issue by jackbaty** [am1t/blotpub#1]
Amit, thanks so much for this service! I had mine up and running (via Heroku) in like 10 minutes.

The frontmatter in Blotpub-created posts consists only of `title: `. For me, the default post then does not show the title in my Blot blog's template, but it _does_ use the title when displaying the post in Micro.blog. I can't tell why this is.

I don't use `title:` in the metadata, but rather use a top-level Markdown heading, which works as expected. So, here's what blotpub created as my first post:

```
title : Posting to Blot using Blotpub

This is a test of Amit Gawande's [Blotpub](https://github.com/am1t/blotpub),
a Micropub service for Blot.im blogs. If this works, thanks Amit!
```

This did not display a title...

![notitle](https://user-images.githubusercontent.com/970/43678486-1edfe832-97e2-11e8-9463-f9e9a6fedd6e.png)

But micro.blog recognized the title somehow and rendered this...

![post](https://user-images.githubusercontent.com/970/43678477-ef6fb208-97e1-11e8-8c0b-c5e396c03d45.png)

If I edit the post like so...

```
title : Posting to Blot using Blotpub

# Posting to Blot using Blotpub

This is a test of Amit Gawande's [Blotpub](https://github.com/am1t/blotpub),
a Micropub service for Blot.im blogs. If this works, thanks Amit!
```

... the title is displayed properly in Blot. I can't explain this!

I also include a date in my frontmatter, since I don't trust using the file's modification date. Here's what my usual Blot post would look like:

```
Date: 2018-08-04 12:18
Tags: blot, blogging

# Posting to Blot using Blotpub

This is a test of Amit Gawande's [Blotpub](https://github.com/am1t/blotpub),
a Micropub service for Blot.im blogs. If this works, thanks Amit!
```

Long story short, I wonder if there's a way that the app could be configured to use the user's preferences for frontmatter?

Thanks again, this is great as-is, so feel free to close this as "WONTFIX" :)

**Comment by am1t** [comment]
Thank you, Jack, for using the app and giving your feedback. A couple of points in response. 

On title not being shown on your blog, it is the preference set by your Blot theme. I think you are using the "default" theme which does not show title (not sure why David kept it like that). If you go to `\archives`, your would see that the title is there. So either try using a different theme or create a new theme from default and edit `entries.html` & `entry.html` to include `{{title}}`.

I am working on bringing in the remaining [supported properties](https://blot.im/help/metadata) to metadata, including the date, over the time.

**Comment by jackbaty** [comment]
Thanks for the clarification. Looking forward to watching where this goes!

**Comment by am1t** [comment]
@jackbaty Just to update you, setting `date` in metadata  [is supported](https://blog.amitgawande.com/blotpub-supports-like-replies) now. With this, all the properties that can be set via Quill are addressed. I will keep this issue open to track possibility of supporting other properties - mainly permalink, summary and draft.

**Comment by am1t** [comment]
Closing the issue as all planned properties are supported now
